### DISCUSSION TITLE
Add `base64URL` as encode / decode format

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#1016] [Ideas]
The [`encode`](https://docs.arcadedb.com/#_encode) and [`decode`](https://docs.arcadedb.com/#_decode) SQL functions only support `base64` encoding. The problem with that is that base64 uses characters that have meaning in URLs, ie `+` and `/`. A derived encoding [`base64URL`](https://base64.guru/standards/base64url) replaces `+` with `-`, and `/` with `_`, which are both characters without need for url encoding. Thus I would like to propose to add base64URL as a format.

I think this shouldn't be too hard, as in the resulting encoding or passed encoded string the characters in question would just have to be replaced before decoding or after encoding with `base64`.

**Comment by lvca** [comment]
Thanks @gramian to support this on https://github.com/ArcadeData/arcadedb/pull/1019. Closing this discussion.
